controllers/user.js

```diff
@@ -0,0 +1,74 @@
+const {User} = require('../models/')
+class ControllerUser {
+    static getAllUser(req,res){
+        User
+        .findAll()
+        .then(users=>{
```

Comment:
Indentasinya harus diperhatikan lagi

```suggestion
        .then(users => {
```